screens/AddCryptoScreen/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC, useState } from 'react';
+import { Text, View, SafeAreaView, TouchableOpacity, TextInput } from 'react-native';
+import { useDispatch } from 'react-redux';
+import styles from './styles';
+import { addCripto } from '../../store/actions/crypto.actions';
+import colors from '../../assets/colors/colors';
+
+interface Props {
+    navigation: any,
+}
+
+const AddCryptoScreen: FC<Props> = ({ navigation }) => {
+
+    const dispatch = useDispatch()
+    const [textInput, setTextInput] = useState<string>('');
+    const [focus, setFocus] = useState<boolean>(false)
+
+    const handlerAddCoin = () => {
+        dispatch(addCripto(textInput))
+        navigation.navigate('Home')
+    }
+
+    const handleFocus = () => setFocus(true)
+
+    return (
+        <SafeAreaView>
+            <View style={styles.addCryptoContainer}>
+                <TouchableOpacity
+                    onPress={() => navigation.navigate('Home')}>
+                    <Text style={styles.backText}>{`< Back to list`}</Text>
```

Comment:
[Minor] delete the {} and ` `, and replace < with a html entity
https://www.w3schools.com/html/html_entities.asp